src/fabric/api/fixed-asset-base/go/main.go

```diff
@@ -122,6 +122,25 @@ func (t *Chaincode) Invoke(stub shim.ChaincodeStubInterface) pb.Response {
 			return shim.Error(err.Error())
 		}
 
+		return shim.Success(bytes)
+	case "readwriteassets":
+		ruuids := []string{}
+		err := json.Unmarshal([]byte(args[0]), &ruuids)
+		wuuids := []string{}
+		err = json.Unmarshal([]byte(args[1]), &wuuids)
+
+		assets, err := fac.ReadWriteAssets(ctx, ruuids, wuuids, args[2])
```

Comment:
I wasn't envisaging that we would return anything as it generally doesn't make sense for a transaction you would commit to return data so I think we can remove returning anything for now or make it optional